react.md

```diff
@@ -4,55 +4,54 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. Here is a list of pros and cons to using the React library to build your application -- but some of them are false. Remove the false statements from the list:
 
-- React was created to be simple, so that even people with minimal code experience could use it and create Single Page Applications (SPAs)
 - React is a modern, efficient answer to complex UI applications
 - React is a full stack framework for modern web applications
 - React is a flexible library that plays the role of V in an MVC framework
 
- 
+
  #### 2. What are "smart"(logic) and "dumb"(display) components? Explain the difference and also add why we bother to make the distinction between them.
- 
- 
- //Your Answer
- 
- 
+
+//Your Answer
+"Smart" logic components are the ones we use in a react application to store our logic such as functions and our state.  These are most often the parent elements
+
+"Dumb" display components are typically the children elements that don't contain logic (or have very minimal).  These components are usually just used to call functions from parent elements
+
  //Googled Answer
- 
- 
+
+
 #### 3. When we use "yarn add ..." in the terminal - what is yarn doing? And what file will always be automatically updated after we add a package with yarn?
- 
- 
+
+
  //Your Answer
- 
- 
+
+
  //Googled Answer
- 
- 
+Yarn add ... will install a package and any packages that it depends on.
+
 #### 5. There are three mistakes in this code that would cause it to break our application. Find the mistakes and fix them:
 
-    import React, { Component } from 'react';
+    import React, { Component } from './react';
```

Comment:
Have another look at this. The first line was good as it was. Hint: Check syntax of the component definition (the class/the render). Also, you were close with state. After revising state in your way, how should the map change?